pcsx2/DEV9/ConfigUI.cpp

```diff
@@ -184,16 +184,15 @@ class DEV9Dialog : public wxDialog
 		hdd_grid->Add(m_hdd_file,     wxSizerFlags().Expand());
 		auto* hdd_size_label = new wxStaticText(this, wxID_ANY, _("HDD Size (GiB):"));
 		auto* hdd_size_box = new wxBoxSizer(wxHORIZONTAL);
-		m_hdd_size_slider = new wxSlider(this, wxID_ANY, HDD_MIN_GB, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_MIN_MAX_LABELS | wxSL_AUTOTICKS);
+		m_hdd_size_slider = new wxSlider(this, wxID_ANY, 10, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_AUTOTICKS);
 		m_hdd_size_slider->SetPageSize(10);
-		for (int i = 15; i < HDD_MAX_GB; i += 5)
-			m_hdd_size_slider->SetTick(i);
+		m_hdd_size_slider->SetTickFreq(10);
 		m_hdd_size_spin = new wxSpinCtrl(this, wxID_ANY, wxEmptyString, wxDefaultPosition, wxDefaultSize, wxSP_ARROW_KEYS, HDD_MIN_GB, HDD_MAX_GB, HDD_MIN_GB);
 		m_hdd_size_spin->SetSizeHints(m_hdd_size_spin->GetSizeFromTextSize(30));
-		hdd_size_box->Add(m_hdd_size_slider, wxSizerFlags(1).Centre());
-		hdd_size_box->Add(m_hdd_size_spin,   wxSizerFlags(0).Centre());
-		hdd_grid->Add(hdd_size_label, wxSizerFlags().Centre().Right());
-		hdd_grid->Add(hdd_size_box,   wxSizerFlags().Expand());
+		hdd_size_box->Add(m_hdd_size_slider, wxSizerFlags(4).Centre().Border(wxLEFT,4)); 
+		hdd_size_box->Add(m_hdd_size_spin,   wxSizerFlags(1).Centre().Border(wxLEFT,4)); 
+		hdd_grid->Add(hdd_size_label, wxSizerFlags().Centre().Right().Border(wxLEFT,4)); 
+		hdd_grid->Add(hdd_size_box,   wxSizerFlags().Expand().Border(wxLEFT,4)); 
```

Comment:
Please don't apply left borders to everything.  If you wanted to add space between the slider and spin control, you can `hdd_size_box->AddSpacer(5);` between the two controls.

I also don't see much point in messing with the proportions like that, there's no reason to grow the size box because it never needs to fit more digits than `HDD_MAX_GB`.  If you're finding that it doesn't fit that, adjust the size of the box by adjusting the number in the `GetSizeFromTextSize` call.

Reply:
Probably why Linux/Mac looks so big as I am screwing with the window properties in janky method.